indico_toolkit/metrics/compare_ground_truth.py

```diff
@@ -66,13 +70,23 @@ def _get_recall(self, true_p: int, false_n: int) -> float:
             recall = 0
         return recall
 
-    def _get_base_metrics(self, label: str) -> dict:
+    def _sequences_exact(self, x: dict, y: dict) -> bool:
+        """
+        Boolean return value indicates whether or not seqs are exact
+        """
+        return x["start"] == y["start"] and x["end"] == y["end"]
```

Comment:
Might be better to put in the associations class - same place where sequences_overlap is

Reply:
nvm, I see the comment to add later